cypress/e2e/signIn.cy.js

```diff
@@ -12,23 +17,54 @@ describe('Sign In page', () => {
 
   before(() => {
     cy.task('db:clear');
+  });
+
+  beforeEach(() => {
     cy.task('generateUser').then((generateUser) => {
       user = generateUser;
+      signInPage.visit();
+      cy.registerUser(user.email, user.username, user.password);
     });
   });
 
   it('should provide an ability to log in with existing credentials', () => {
-    signInPage.visit();
-    cy.register(user.email, user.username, user.password);
-
-    signInPage.typeEmail(user.email);
-    signInPage.typePassword(user.password);
-    signInPage.clickSignInBtn();
+    signInPage.enterLoginCredentials(user.email, user.password);
```

Comment:
The `enterLoginCredentials` method is not defined in the SignInPageObject class. Make sure you have it properly defined and imported.